generators/app/templates/.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_VERSION=$npm_package_version
+REACT_APP_NAME=$npm_package_name
+REACT_APP_COMPANY_NAME='inrupt Inc.'
```

Comment:
Shouldn't 'inrupt Inc.' be capitalized?

Reply:
Actually no, inrupt is all lower case according to Tony and our design guide